spec/chartmogul/csv/line_items/one_item_spec.rb

```diff
@@ -54,7 +54,8 @@
           Description
           Transaction\ fee
           Account\ Code
-          Transaction\ fee\ currency
+          Transaction\ currency
```

Comment:
see https://github.com/chartmogul/platform/blob/1ce70c9498cbb4c3f7d993bb3c8f5297a2fd2561/engines/data_ingestion/app/services/data_ingestion/csv_mapping/line_item.rb#L24

Reply:
PS. the CSV header might be confusing. I informed Blake.

Reply:
Isn't that mapping a bug?
The field is called `transaction_fees_currency`... it's not Transaction currency :|

Reply:
Yes. And I fixed it.